test/apiv2/36-quadlets.at

```diff
@@ -0,0 +1,15 @@
+# -*- sh -*-
+#
+# quadlet-related tests
+#
+
+# NOTE: Once podman-remote quadlet support is added we can enable the podman quadlet tests in
+#       test/system/253-podman-quadlet.bats which should cover it in more detail then.
+
+## list volume
+t GET libpod/quadlets/json 200
```

Comment:
I would also check the request's output to verify that the returned JSON contains the expected data.

Reply:
The problem is that the test here cannot make any assumptions about the output, this suite should be independent from you host files and if you happen to have a quadlet file this will return one otherwise by default in CI this will be empty so I don't see a good way to achieve this here. I would really just wait until we have the proper system tests via podman-remote and call this good enough.